projects/plugins/jetpack/_inc/client/components/plugin-dash-item/index.jsx

```diff
@@ -174,15 +174,15 @@ PluginDashItem.propTypes = {
 
```

Comment:
There is a failing prop type: The prop type `pluginFile` is is marked as required, but its value is `undefined`.

I think prop type `pluginFile` should be changed to `pluginFiles: PropTypes.array.isRequired,`

Reply:
Thanks for your review, and catching this @elliottprogrammer! 

I've made a fix via fdc4189. I've opted to use `arrayOf` to keep the specific string type for each plugin file path inside of `pluginFiles`:

```javascript
pluginFiles: PropTypes.arrayOf( PropTypes.string ).isRequired,
```

Reply:
Nice! Even better!  Everything LGTM!